cmd/controller/main.go

```diff
@@ -110,8 +114,8 @@ func main() {
 
 	ctx = filteredinformerfactory.WithSelectors(ctx, v1beta1.ManagedByLabelKey)
 	sharedmain.MainWithConfig(ctx, ControllerLogKey, cfg,
-		taskrun.NewController(*namespace, images),
-		pipelinerun.NewController(*namespace, images),
+		taskrun.NewController(*namespace, images, *experimentalDisableInTreeResolution),
```

Comment:
I know it's sort of a misnomer, but it feels like it'd be nice if this was passed through `images` 🤔 

Since this is a breaking signature change anyways, I wonder whether it's worth renaming the struct and passing this field through it instead.

Reply:
I think I'm missing a bit here - why's that preferable?

Reply:
I've added a struct for each controller to pass the images and disableresolution flag through.